### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by dshkol**
- Rewrite some of the background description and API key information
- Drop tmap example for ggplot2 example
- Add Version History section

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,3 +66,8 @@ tm_shape(census_data) +
 * [Example 1](http://htmlpreview.github.io/?https://raw.githubusercontent.com/mountainMath/cancensus/master/vignettes/cancensus-demo.nb.html)
 * [Example 2](http://htmlpreview.github.io/?https://github.com/mountainMath/cancensus/blob/master/vignettes/Working-with-cancensus.html)
 * [Example 3](http://htmlpreview.github.io/?https://github.com/mountainMath/cancensus/blob/master/vignettes/dot_density_example.nb.html)
+
+## Version History
```
Comment (atheriel) [inline README.md]: We might need to split this out into a separate `NEWS.md` file at some point.

**Comment by atheriel** [comment]
:+1:

File: README.md (hunk 1 above)
Comment (dshkol) [inline README.md]: Can it be in both? I've seen that in both.

Comment (atheriel) [inline README.md]: Of course.

**Comment by atheriel** [comment]
I think this can be merged.

**Pull request merged**